aten/src/ATen/native/cpu/ReduceOpsKernel.cpp

```diff
@@ -124,6 +124,16 @@ static void logcumsumexp_cpu_kernel(Tensor& result, const Tensor& self, int64_t
   });
 }
 
+static void nansum_kernel_impl(TensorIterator& iter) {
```

Comment:
Would you add a "TODO:" comment here to unify this implementation with the implementation of sum in cpu/SumKernel.cpp? It has an involved bespoke implementation.